### ISSUE TITLE
Testsuite failure on master

### ISSUE THREAD

**Issue by andreasabel** [corsis/clock#77]
Seems like the testsuite is broken on `master`:
```
$ cabal test -w ghc-8.10.7
Build profile: -w ghc-8.10.7 -O1
In order, the following will be built (use -v for more details):
 - clock-0.8.2 (test:test) (first run)
Preprocessing test suite 'test' for clock-0.8.2..
Building test suite 'test' for clock-0.8.2..
Running 1 test suites...
Test suite test: RUNNING...
All tests
  Num instance tests
    QuickCheck
      x = abs(x) * signum(x):                                              FAIL
        *** Failed! Falsified (after 2 tests):
        TimeSpec {sec = -1, nsec = 0}
        Use --quickcheck-replay=634946 to reproduce.
        Use -p '/x = abs(x) * signum(x)/' to rerun this test only.
      integer     addition equals TimeSpec     addition:                   OK (0.17s)
        +++ OK, passed 100100 tests.
      integer subtraction equals TimeSpec subtracttion:                    OK (0.14s)
        +++ OK, passed 100100 tests.
      rational multiplication equals TimeSpec multiplication:              FAIL
        *** Failed! Falsified (after 2 tests and 7 shrinks):
        0.010000000
        0.100000000
        Use --quickcheck-replay=18476 to reproduce.
        Use -p '/rational multiplication equals TimeSpec multiplication/' to rerun this test only.
      neg(neg(x)) = x:                                                     OK (0.09s)
        +++ OK, passed 100100 tests.
  Ord instance tests
    QuickCheck
      random list of TimeSpecs is sorted like equivalent list of integers: OK (2.26s)
        +++ OK, passed 100100 tests.

2 out of 6 tests failed (2.69s)

Test suite test: FAIL
```

**Comment by cetinsert** [comment]
@andreasabel – any suggestions on how to address this?

**Comment by andreasabel** [comment]
Well, you already fixed this by merging #69.
